README.md

```diff
@@ -1,26 +1,97 @@
-# sumologic-public-partner-apps
```

Comment:
We need to update the manifest.json https://github.com/SumoLogic/sumologic-public-partner-apps/blob/master/Nucleon/Nucleon.manifest.json#L9

and update the helpURL field to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Nucleon